contracts/gateway/GatewayBase.sol

```diff
@@ -67,12 +68,12 @@ contract GatewayBase is Organized {
     /* Constants */
 
     /** Position of message bus in the storage. */
-    uint8 constant MESSAGE_BOX_OFFSET = 7;
+    uint8 constant public MESSAGE_BOX_OFFSET = 9;
```

Comment:
In the file some places it's `public constant` and here it's `constant public`, not sure which is correct.

Reply:
We should also add getters for internal and private mapping?